client/components/Message/Actions/Action.tsx

```diff
@@ -27,8 +27,19 @@ const Action: FC<ActionOptions> = ({ id, icon, i18nLabel, label, mid, runAction
 
 	const resolvedIcon = resolveLegacyIcon(icon);
 
+	const dangerButton = label === t('End_call');
```

Comment:
I don't think its a good practice to resolve the value of this property here since this is a general component. Ideally it should be passed within the props just like other properties like `icon`, `label`, etc are being passed.

@gabriellsh You have more experience with UI components, am I correct in the above?